Controllers/Backend/MolliePayments.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Mollie\Api\MollieApiClient;
+use MollieShopware\Components\ApplePayDirect\Services\ApplePayDomainFileDownloader;
+use MollieShopware\Components\Config;
+use MollieShopware\Components\MollieApiFactory;
+use MollieShopware\Components\Services\PaymentMethodService;
+use MollieShopware\Components\Services\ShopService;
+use Psr\Log\LoggerInterface;
+use Shopware\Components\Model\ModelManager;
+use Shopware\Components\Plugin\ConfigReader;
+use Shopware\Components\Plugin\PaymentInstaller;
+
+class Shopware_Controllers_Backend_MolliePayments extends Shopware_Controllers_Backend_ExtJs
+{
+    public function updateAction()
+    {
+        try {
+            /** @var PaymentMethodService $paymentMethodService */
+            $paymentMethodService = $this->container->get('mollie_shopware.payment_method_service');
```

Comment:
please use mollie logger with 
INFO: Importing Payment Methods
....
and ERROR in catch